### ISSUE TITLE
Latest release flag points to previous v17.03.2-ce

### ISSUE THREAD

**Issue by luislavena** [boot2docker/boot2docker#1266]
Hello,

When using `docker-machine upgrade` to upgrade a VirtualBox VM that uses boot2docker, it is pulling the wrong version.

Seems that it uses the *latest* release tagged on GitHub: https://github.com/boot2docker/boot2docker/releases/latest

```
$ curl -I https://github.com/boot2docker/boot2docker/releases/latest | grep Location
Location: https://github.com/boot2docker/boot2docker/releases/tag/v17.03.2-ce
```

Which results pointing to `v17.03.2-ce` instead of most recent `v17.06.0-ce`

To workaround this issue I've manually upgraded my local `.iso` image on both `~/.docker/cache` and within the machine itself.

However, perhaps there is something that can be done within GitHub settings to point to latest release?

Thank you. ❤️ ❤️ ❤️

**Comment by yosifkit** [comment]
I got this same issue in a chat; I was able to fix this by changing the git tag date of `v17.03.2-ce` to be older than `v17.06.0-ce`.  Let me know if it working now for you. :smile:

**Comment by luislavena** [comment]
Hello!

It appears to be working! I've removed the `machine/cache` ISO and then performed the upgrade:

```
No default Boot2Docker ISO found locally, downloading the latest release...
Latest release for github.com/boot2docker/boot2docker is v17.06.0-ce
```

Thank you! ❤️ ❤️ ❤️
